edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -170,7 +168,6 @@ def _decode_and_verify_token(token, jwt_issuer):
     token_issuer = decoded_token.get('iss')
     issuer_matched = any(issuer['ISSUER'] == token_issuer for issuer in get_jwt_issuers())
     if not issuer_matched:
-        logger.info('Token decode failed due to mismatched issuer [%s]', token_issuer)
```

Comment:
@nasthagiri: As noted in Slack, I could leave this info log if you think it is important, but I would imagine that the raised InvalidTokenError would get logged.  Same goes for the above info log message.  I'm good with leaving it or removing it.

Reply:
Left as-is.